packages/cozy-mespapiers-lib/src/components/Search/search.js

```diff
@@ -37,8 +37,7 @@ export const index = new Document({
       'civility', // io.cozy.contacts
       'company', // io.cozy.contacts
       'jobTitle' // io.cozy.contacts
-    ],
-    store: ['_id', '_type']
```

Comment:
Don't we need the `_type` do differentiate contacts from files?

Reply:
non finalement ça ne sert à rien, on part du principe que les docs sont uniques tout doctype confondu, et on a tous les doc tout doctype confondu dans un seul object au final.

Reply:
Quand tu requêtes cozy-client, tu as besoin de connaître le type pour savoir quelle bd requêter

Reply:
on ne fait pas de requête à CC pour avoir le doc, on a déjà tous les doc en amont, et on fait un matching dans un object qui contient déjà tous les docs. Pour l'instant tout est synchrone. Si besoin de changer plus tard, on changera :)